### DISCUSSION TITLE
cn/2024/03/23/discrete-distributions/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#157] [Announcements]
# cn/2024/03/23/discrete-distributions/

https://hongtaoh.com/cn/2024/03/23/discrete-distributions/

**Comment by wangyunzi** [comment]
你这个排版真的很好看很清楚

**Reply by hongtaoh** [reply]
难得有人夸我的数学博客！很感谢。我自己反而觉得排版不太清楚，有侧栏目录更好，但我不想花时间弄了。
